apps/ledger-live-desktop/src/renderer/components/TransactionConfirm/ConfirmFooter.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import styled from "styled-components";
+import { Trans, withTranslation } from "react-i18next";
+import { getEnv } from "@ledgerhq/live-env";
+import Text from "~/renderer/components/Text";
+import { openURL } from "~/renderer/linking";
+
+const HorizontalSeparator = styled.div`
+  height: 1px;
+  background: ${p => p.theme.colors.palette.text.shade20};
+  width: 100%;
+`;
+
+const termsOfUse = new Map<string, string>([
+  ["paraswap", "https://paraswap.io/tos"],
+  ["1inch", "https://1inch.io/assets/1inch_network_terms_of_use.pdf"],
+]);
+
+if (getEnv("PLAYWRIGHT_RUN")) {
+  termsOfUse.set("dummy-live-app", "https://localhost.io/testtos");
+}
+
+type Props = {
+  footer: React.ReactNode | undefined;
+  manifestId?: string | null;
+  manifestName?: string | null;
+};
+
+const ConfirmFooter = ({ footer, manifestId, manifestName }: Props) => {
+  if (manifestId) {
```

Comment:
This might be a bit cleaner to instead do an early return when there is no `manifestId` and same for `termsOfUseUrl` too